indicators/management/commands/create_participant_count_indicators.py

```diff
@@ -0,0 +1,230 @@
+                metrics['dates_created'] += 1
+            try:
+                top_level = Level.objects.get(parent=None, program=dummy_program)
+            except Level.DoesNotExist:
+                top_level = Level.objects.create(name='Goal', customsort=1, program=dummy_program)
+                for i, tier_name in enumerate(LevelTier.get_templates()['mc_standard']['tiers']):
+                    LevelTier.objects.create(name=tier_name, program=dummy_program, tier_depth=i+1)
+                metrics['rf_created'] += 1
+
+            for i in range(indicator_count):
+                indicator_name = 'Number of people reached in fiscal year in <<insert program name>>'
+                i_created = self.create_indicator(indicator_name, dummy_program, top_level, 3, pc_country)
+                if i_created:
+                    try:
+                        metrics['indicator_created'][country] += 1
+                    except KeyError:
+                        metrics['indicator_created'][country] = 1
+                else:
+                    try:
+                        metrics['indicator_existed'][country] += 1
+                    except KeyError:
+                        metrics['indicator_existed'][country] = 1
+
+
+        print(f"{metrics['program_created']} programs were created in the dummy country, {metrics['program_existed']} already existed.")
+        print(f"{metrics['dates_created']} reporting period dates were created.")
+        print(f"{metrics['rf_created']} programs needed a result framework level added.")
+        for country_name in metrics['indicator_created']:
+            print(f"{metrics['indicator_created'][country_name]} indicators were created in {country_name}.")
+        for country_name in metrics['indicator_existed']:
+            print(f"{metrics['indicator_existed'][country_name]} indicators already existed in {country_name}.")
+
+    @staticmethod
+    def create_indicator(name, program, top_level, option_number, country):
+        definition_text = (
+            "Participants are defined as “all people who have received tangible benefit – directly "
+            "or indirectly from the project.” We distinguish between direct and indirect:\n\n"
+            "Direct participants – are those who have received a tangible benefit from the program, "
+            "either as the actual program participants or the intended recipients of the program benefits. "
+            "This means individuals or communities.\n\n"
+            "Indirect participants – are those who received a tangible benefit through their proximity to "
+            "or contact with program participants or activities.\n\n"
+            "** Add Direct Participants definition **\n\n"
+            "** Add Indirect Participants definition **"
+        )
+        indicator_justification = (
+            "Participant reach is crucial to take decisions on the program implementation.It provides insights "
```

Comment:
Should this me "make" instead of "take"? Also, missing a space between sentences.

Reply:
The copy was given to me so I'll just leave it.  But I'll fix the missing space, thanks!